cmd/legacy_main.go

```diff
@@ -297,18 +321,10 @@ func runCLIApp(c *cli.Context) (err error) {
 	// if these are already being logged into a log-file, otherwise
 	// there will be duplicate logs for these in both places (stdout and log-file).
 	if newConfig.Foreground || newConfig.Logging.FilePath == "" {
-		flagsStringified, err := util.Stringify(*flags)
-		if err != nil {
-			logger.Warnf("failed to stringify cli flags: %v", err)
-		} else {
-			logger.Infof("GCSFuse mount command flags: %s", flagsStringified)
-		}
-
-		mountConfigStringified, err := util.Stringify(*mountConfig)
-		if err != nil {
-			logger.Warnf("failed to stringify config-file: %v", err)
+		if cfg.IsNewConfigEnabled() {
+			logNewConfiguration(newConfig)
```

Comment:
Shall we log new config object regardless as that is the final one being used in the code (eventually)

Reply:
It does make sense, but decided against it because with the recent change, we are logging a json object instead of a plain string. I'd like to see how it works with Pantheon logging before turning it on by default. Another minor concern was that it'd add an extra log :)

Reply:
We can check how it works with pantheon logging using https://cloud.google.com/logging/docs/agent/logging/installation.

> I didn't want to add more logs to log configuration. We are already logging the config and flag and this will add more duplicate information.

I was thinking about just logging the new config.